yt_dlp/extractor/tnaflix.py

```diff
@@ -234,6 +234,7 @@ class TNAFlixIE(TNAEMPFlixBaseIE):
             'display_id': 'Carmella-Decesare-striptease',
             'ext': 'mp4',
             'title': 'Carmella Decesare - striptease',
+            'description': 'No description provided',
```

Comment:
the site actually returns this?

Reply:
afraid so- I just dumped a random # from `--test`